config.js

```diff
@@ -84,11 +84,11 @@
       evrDefaultBackgroundColor: undefined,
 
       /**
-       * evrDefaultForegoundColor: default foreground color for EVRs.
+       * evrDefaultForegroundColor: default foreground color for EVRs.
        * Set to `undefined` to use the theme default.  Otherwise, specify
        * a hex string for an RGB color, e.g. `#ababab`.
        */
-      evrDefaultForegoundColor: undefined,
+      evrDefaultForegroundColor: undefined,
```

Comment:
This may pop up some issues on copy/paste config transitions, just something to keep in the back of our minds.

Reply:
true. we should probably add this to release notes